library/kotlin/src/io/envoyproxy/envoymobile/Client.kt

```diff
@@ -0,0 +1,12 @@
+package io.envoyproxy.envoymobile
+
+interface Client {
+  /**
+   * For starting a stream
+   *
+   * @param request the request for opening a stream
+   * @param streamCallback the callback for receiving stream events
+   * @return the emitter for streaming data outward
+   */
+  fun startStream(request: Request, streamCallback: StreamCallback): StreamEmitter
```

Comment:
What do you think about renaming this to simply `start`?

Reply:
Also think `streamCallback` can simply be `callback`

Reply:
The unary callback will be called `UnaryCallback` then?

Totally fine with this

Reply:
Discussed offline, let's leave streaming as the first-class citizen here